src/prng.h

```diff
@@ -24,23 +24,23 @@
 /* A chunk of random data. */
 typedef struct /* nwipe_entropy_t */
 {
-	size_t  length;  /* Length of the entropy string in bytes.  */
-	u8*     s;       /* The actual bytes of the entropy string. */
+    size_t length; /* Length of the entropy string in bytes.  */
+    u8*    s;      /* The actual bytes of the entropy string. */
 } nwipe_entropy_t;
 
-#define NWIPE_PRNG_INIT_SIGNATURE void** state, nwipe_entropy_t* seed
-#define NWIPE_PRNG_READ_SIGNATURE void** state, void* buffer, size_t count
+#define NWIPE_PRNG_INIT_SIGNATURE void **state, nwipe_entropy_t *seed
```

Comment:
@PartialVolume do you think this might cause some problems? I don't understand why the pointers are right aligned only in this case. We can always exclude those defines from formatting.

Reply:
I'm not sure what you mean. Do you mean the void** state as opposed to the void **state. I don't think it makes any difference, although to be sure I would probably write a small program to test it.

Reply:
In GUI.h the pointers align from originally right to left in the new format, which is different to above. Seems a bit odd, I wonder if there is a reason it justifies left in one file and right in another file.

Reply:
See [https://stackoverflow.com/questions/38392889/clang-format-align-asterisk-of-pointer-declaration-with-variable-name](https://stackoverflow.com/questions/38392889/clang-format-align-asterisk-of-pointer-declaration-with-variable-name)

Maybe it's not implemented properly.

Reply:
yeah it is strange. The format is configured to align left, but looks like it does the opposite on `#define` statements. It might be a bug in `clang-format`. We can exclude those statements from formatting if needed.